modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-deploy.adoc

```diff
@@ -74,6 +72,8 @@ helm upgrade --install redpanda-controller redpanda/operator \
   --set image.repository=docker.redpanda.com/redpandadata/redpanda-operator \
   --create-namespace
 ----
++
+NOTE: If you already have Flux installed on your Kubernetes cluster, use the `--set enableHelmControllers=false` flag to prevent the Redpanda Operator from deploying its own set of Helm controllers.
```

Comment:
```suggestion
NOTE: If you already have Flux installed on your Kubernetes cluster and wish it to continue managing your entire cluster, use the `--set enableHelmControllers=false` flag to prevent the Redpanda Operator from deploying its own set of Helm controllers that may conflict with those installed with https://fluxcd.io/flux/concepts/[Flux^].
```